### PR TITLE
Trim codeowners

### PR DESCRIPTION
**Pull request by brokensound77**
## Summary

Trim codeowners

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-tests/**/*.py     @brokensound77 @mikaayenson @eric-forte-elastic
-detection_rules/  @brokensound77 @mikaayenson @eric-forte-elastic
-tests/            @brokensound77 @mikaayenson @eric-forte-elastic
-lib/              @brokensound77 @mikaayenson @eric-forte-elastic
-rta/              @brokensound77 @mikaayenson @eric-forte-elastic
+tests/**/*.py     @mikaayenson @eric-forte-elastic
+detection_rules/  @mikaayenson @eric-forte-elastic
+tests/            @mikaayenson @eric-forte-elastic
+lib/              @mikaayenson @eric-forte-elastic
+rta/              @mikaayenson @eric-forte-elastic
```
Comment (Mikaayenson) [inline .github/CODEOWNERS:8]:
```suggestion
tests/**/*.py     @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/  @mikaayenson @eric-forte-elastic @terrancedejesus 
tests/            @mikaayenson @eric-forte-elastic @terrancedejesus 
lib/              @mikaayenson @eric-forte-elastic @terrancedejesus 
rta/              @mikaayenson @eric-forte-elastic @terrancedejesus 
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 # skip rta-mapping to avoid the spam
-detection_rules/etc/packages.yaml  @brokensound77 @mikaayenson @eric-forte-elastic
-detection_rules/etc/*.json        @brokensound77 @mikaayenson @eric-forte-elastic
-detection_rules/etc/*.json        @brokensound77 @mikaayenson @eric-forte-elastic
-detection_rules/etc/*/*           @brokensound77 @mikaayenson @eric-forte-elastic
-
+detection_rules/etc/packages.yaml @mikaayenson @eric-forte-elastic
+detection_rules/etc/*.json        @mikaayenson @eric-forte-elastic
+detection_rules/etc/*.json        @mikaayenson @eric-forte-elastic
+detection_rules/etc/*/*           @mikaayenson @eric-forte-elastic
```
Comment (Mikaayenson) [inline .github/CODEOWNERS:14]:
```suggestion
detection_rules/etc/packages.yaml @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/etc/*.json        @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/etc/*.json        @mikaayenson @eric-forte-elastic @terrancedejesus 
detection_rules/etc/*/*           @mikaayenson @eric-forte-elastic @terrancedejesus 
```

**Review by terrancedejesus — APPROVED** [review]
Looks good.

**Review by shashank-elastic — APPROVED** [review]
Approving provided we have Terrance added as per suggestion to codeowners

**Pull request merged**